explainaboard/third_party/README.md

```diff
@@ -0,0 +1,32 @@
+# Third-party libraries
+
+This directory contains libraries manually imported from other place.
+Code in this directory is maintained almost under the *as-is* basis. We don't guarantee
+that code is maintained under the same maintenance policy we applied to other parts in
+this repository.
+We also don't guarantee that the code is up-to-date.
+
+Before considering to upload any code to this directory, you have to make sure that the
+change satisfies **all** of following conditions:
+
+- If the code is trivial enough, consider implementing a similar logic to our main
+  component.
+- Code is distributed under a valid open-source license. The license have to be
+  compatible with the MIT license, which we applied to this repository.
+- Code is not able to be obtained from the PyPI.
```

Comment:
> The license have to be compatible with the MIT license
* `have` -> `has`
* could we elaborate more on `compatible`? For example, how do contributors know whether their code is compatible with MIT?

Reply:
Thanks, I added some ways to judge the compatibility.